apps/web/src/stores/editor-store.js

```diff
@@ -145,6 +147,10 @@ class EditorStore extends BaseStore {
         attachmentsLength: db.attachments.ofNote(note.id, "all")?.length || 0
       };
     });
+    if (!db.notes.note(noteId))
+      this.set((state) => {
```

Comment:
There's no need to add a separate call here. You can just update the state in a single `this.set` call which is being made in the previous lines.